src/main/kotlin/no/nav/familie/ef/sak/api/beregning/Beregning.kt

```diff
@@ -1,11 +1,34 @@
 package no.nav.familie.ef.sak.api.beregning
 
+import no.nav.familie.ef.sak.util.isEqualOrBefore
 import java.math.BigDecimal
 import java.time.LocalDate
 
 data class Beløpsperiode(val fraOgMedDato: LocalDate,
                          val tilDato: LocalDate,
-                         val beløp: BigDecimal)
+                         val beregningsgrunnlag: Beregningsgrunnlag? = null,
+                         val beløp: BigDecimal) {
```

Comment:
Det er iallefall to variabler som kommer til å være "nyttig" å få ut av denne i tillegg til de som finnes fra før;
- `beløpFørSamordning` (i praksis `beløp + samordningsfradrag`, men liker ikke matte i javascript) 
- `inntektsreduksjon` => Bittelitt usikkerpå eksakt verdi her, men tror det er det samme som `avkortning` i utregningen. Det er ikverksettingsgjengen som har det på agendaen, iallefall.

Reply:
Endret til ```
data class Beregningsgrunnlag(val inntekt: BigDecimal,
                              val samordningsfradrag: BigDecimal,
                              val avkortningPerMåned: BigDecimal,
                              val fullOvergangsStønadPerMåned: BigDecimal,
                              val grunnbeløp: BigDecimal)
```

Reply:
Men kanske rart att visse verdier er perMåned mens andre er for ett år?

Reply:
Kanskje, men tror alle "resultat"-verdiene er ønskelig å få på månedsbasis